edb/schema/functions.py

```diff
@@ -1152,7 +1224,7 @@ class RenameFunction(sd.RenameObject, FunctionCommand):
     pass
 
 
-class AlterFunction(sd.AlterObject, FunctionCommand):
+class AlterFunction(sd.AlterObject[Function], FunctionCommand):
```

Comment:
`AlterCallableObject[Function]`.   Also, `AlterOperator` should inherit from `AlterCallableObject` as well.